migration/migrating_3_4/migrating-application-workloads-3-to-4.adoc

```diff
@@ -6,9 +6,9 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the Cluster Application Migration (CAM) tool. The CAM tool enables you to control the migration and to minimize application downtime.
+You can migrate application workloads from {product-title} 3.7 (and later) to {product-title} {product-version} with the {mtc-first}. The {mtc} enables you to control the migration and to minimize application downtime.
 
-The CAM tool's web console and API, based on Kubernetes Custom Resources, enable you to migrate stateful application workloads at the granularity of a namespace.
+The {mtc}'s web console and API, based on Kubernetes Custom Resources, enable you to migrate stateful application workloads at the granularity of a namespace.
```

Comment:
appears as {mtc}

Reply:
Fixed.

If you see an error like this, check the attributes